.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
+	<classpathentry kind="lib" path="/home/pw/Downloads/java-getopt-1.0.13.jar"/>
```

Comment:
I guess this does not work in general ;-)
I suggest to download that package and integrate it into a subfolder of this project. Of course, you need to check if this is permitted by licensing.